s3-cloudfront-spa/template.tf

```diff
@@ -0,0 +1,145 @@
+terraform {
+    backend "s3" {
+      key = "terraform.tfstate"
+      region = "eu-west-2"
+      bucket = "my-projects-terraform-state"
```

Comment:
As bucket names must be unique, this will lead to problems.

Reply:
Remove the Terraform remote setup. It is not mandatory for the pattern.